app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -575,12 +575,20 @@ const SendBitcoinDetailsScreen = ({
                 <View style={Styles.walletSelectorBalanceContainer}>
                   {sendingWalletDescriptor.currency === WalletCurrency.Btc ? (
                     <>
-                      <Text style={Styles.walletBalanceText}>
+                      <Text
+                        style={Styles.walletBalanceText}
+                        {...testProps("BTC Wallet Balance in USD")}
+                      >
                         {typeof btcBalanceInUsd === "number"
                           ? formatToDisplayCurrency(btcBalanceInUsd) + " - "
                           : ""}
+                      </Text>
+                      <Text
+                        style={Styles.walletBalanceText}
+                        {...testProps("BTC Wallet Balance in sats")}
+                      >
                         {typeof btcWalletBalance === "number"
-                          ? Number(satAmountDisplay(btcWalletBalance))
+                          ? satAmountDisplay(btcWalletBalance)
```

Comment:
why is this fixing the issue?

Reply:
`satAmountDisplay` returns a string which can't be converted to a number because it returns the value + "sats" https://github.com/GaloyMoney/galoy-mobile/blob/23b92256af045d7862da6667e6aaf73d12e606dd/app/utils/currencyConversion.ts#L117